authui/src/error.ts

```diff
@@ -1,9 +1,16 @@
 import { AxiosResponse } from "axios";
+import { session } from "@hotwired/turbo";
 
 export function handleAxiosError(e: unknown) {
   const err = e as any;
   if (err.response != null) {
     const response: AxiosResponse = err.response;
+
+    if (typeof response.data === "string") {
+      session.renderStreamMessage(response.data);
+      return;
+    }
+
     const reason = response.data?.error.reason;
```

Comment:
We can now remove the special handling of `"RateLimited"` error. Just show `error-message-server`. And we can remove the hidden `error-message-rate-limited` from `__error.html`.